repos/spack_repo/builtin/packages/relion/package.py

```diff
@@ -17,11 +17,12 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "https://www2.mrc-lmb.cam.ac.uk/relion"
     git = "https://github.com/3dem/relion.git"
     url = "https://github.com/3dem/relion/archive/4.0.0.zip"
-    maintainers("dacolombo")
+    maintainers("Markus92")
```

Comment:
It's probably better to leave the other maintainer listed unless they say they want to be removed

```suggestion
    maintainers("dacolombo", "Markus92")
```